packages/grafana-ui/src/components/Link/Link.tsx

```diff
@@ -1,5 +1,5 @@
 import { AnchorHTMLAttributes, forwardRef } from 'react';
-import { Link as RouterLink } from 'react-router-dom';
+import { Link as RouterLink } from 'react-router-dom-v5-compat';
```

Comment:
Does grafana/ui have this package as a dependency?

Reply:
Good catch! Will replace `react-router-dom` with this package.